rdrf/rdrf/auth/forms.py

```diff
@@ -139,8 +139,8 @@ def save(self, commit=True):
                     self.user.is_active = True
             else:
                 logger.warning(
-                    'User "%s" resetted their password but their account is inactive '
-                    'and settings.ACCOUNT_SELF_UNLOCK_ENABLED is NOT set.', self.user)
+                    'User id "%s" resetted their password but their account is inactive '
+                    'and settings.ACCOUNT_SELF_UNLOCK_ENABLED is NOT set.', self.user.id)
```

Comment:
For Midland they are actually interested in knowing who did what so I think recording the username is ok - it will be difficult for them to find the corresponding user from the id. For patients , it is different, ids are good in that case.

Reply:
no problem I'll make the change.